components/pages/WalletSettings/OtherWallet.tsx

```diff
@@ -0,0 +1,61 @@
+import Link from "public/assets/icons/link.svg";
+import styled from "styled-components";
+import { AllowedAction } from "./AllowedAction";
+import { AddressWrapper, BarButtonSecondary, BarWrapper, Header, Text } from "./styles";
+
+export function OtherWallet({
+  status,
+  address,
+  remove,
+}: {
+  status: "delegator" | "delegate";
+  address: string | undefined;
+  remove: (address: string) => void;
+}) {
+  if (!address) return null;
+
+  const text =
+    status === "delegate"
+      ? "Explanation of secondary wallet when secondary is delegate."
+      : "Explanation of secondary wallet when secondary is delegator.";
+
+  return (
+    <>
+      <Header>Other wallet ({status})</Header>
```

Comment:
nit: this shows the status in lower case while `Connected Wallet (Delegate/Delegator)` has the first letter in upper case